src/rgw/rgw_rest_swift.h

```diff
@@ -43,7 +43,7 @@ class RGWListBuckets_ObjStore_SWIFT : public RGWListBuckets_ObjStore {
     return 0;
   }
 public:
-  RGWListBuckets_ObjStore_SWIFT() : need_stats(true) {}
+  RGWListBuckets_ObjStore_SWIFT() : need_stats(false) {}
```

Comment:
@rzarzynski Thanks for the quick review. I was referring to http://tracker.ceph.com/issues/11285 where this check `need_stats = (s->format != RGW_FORMAT_PLAIN)` was removed. 

With this patch this is what I see for my curl queries, am I missing anything here.

```
curl -i 'http://localhost:8000/swift/v1?stats=true&format=json' -X GET -H "X-Auth-Token: "
HTTP/1.1 200 OK
X-Timestamp: 1498141604.33927
X-Account-Container-Count: 1
X-Account-Object-Count: 1
X-Account-Bytes-Used: 5872
X-Account-Bytes-Used-Actual: 8192
X-Account-Access-Control: {}
X-Trans-Id: tx0000000000000000037c1-00594bd3a4-101b-default
X-Openstack-Request-Id: tx0000000000000000037c1-00594bd3a4-101b-default
Content-Type: application/json; charset=utf-8
Content-Length: 40
Date: Thu, 22 Jun 2017 14:26:44 GMT


curl -i 'http://localhost:8000/swift/v1?stats=true' -X GET -H "X-Auth-Token: "
HTTP/1.1 200 OK
X-Timestamp: 1498141609.86924
X-Account-Container-Count: 1
X-Account-Object-Count: 1
X-Account-Bytes-Used: 5872
X-Account-Bytes-Used-Actual: 8192
X-Account-Access-Control: {}
X-Trans-Id: tx0000000000000000037c2-00594bd3a9-101b-default
X-Openstack-Request-Id: tx0000000000000000037c2-00594bd3a9-101b-default
Content-Type: text/plain; charset=utf-8
Content-Length: 4
Date: Thu, 22 Jun 2017 14:26:49 GMT
```